inc/head.php

```diff
@@ -165,8 +165,12 @@ function loadKeyboardFromHash() {
 
 <script type='text/javascript'>
  var pageLoading = true;
- <?=$kmwroot?>.init({attachType:'auto'}); //{resources:'resources'});
- <?=$kmwroot?>.addKeyboards();
+ <?=$kmwroot?>.init({
+    attachType:'auto',
+    //{resources:'resources'});
+  }).then(function() {
+    <?=$kmwroot?>.addKeyboards();
+  }); 
```

Comment:
We should always use `keyman.` instead of `<?$kmwroot?>.` as we are never going to support versions `<` 10 any more, and in any case, when did `init()` start supporting promises?

```suggestion
 keyman.init({
   attachType:'auto'
 }).then(function() {
   keyman.addKeyboards();
 }); 
```

So then, we should also remove the definition of `$kmwroot` from ll 31-35, and while we are at it, get rid of the `$version` variable and always assume we are working with >= 10.0 in ll 130-136... 😁